.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+exclude: lib
```

Comment:
I wasn't familiar with what this file was. Looks like it's for the https://pre-commit.com/ tool? Maybe we can describe it briefly here in comments or in the README?

Reply:
Yeh, can we drop this file before merge? I had been running `golang-ci` on the project, and there is already a config file for it :)